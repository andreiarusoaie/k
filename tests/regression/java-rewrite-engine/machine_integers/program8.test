 MyListItem(uvalueMInt(miMInt(usubMInt(mi(31, 0), mi(31, -1))))),
 MyListItem(overflowMInt(usubMInt(mi(31, 0), mi(31, -1))))

