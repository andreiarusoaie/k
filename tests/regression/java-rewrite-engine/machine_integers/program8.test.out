<k>
    MyListItem ( 1 ) , MyListItem ( true )
</k>
